contrib/src/sceneeditor/dataHolder.py

```diff
@@ -737,7 +737,7 @@ def addCollisionObject(self, collisionObj, nodePath, pointA=None, pointB=None, p
         node.addSolid(collisionObj)
         self.collisionDict[name] = nodePath.attachNewNode(node)
 
-        if pointA!=None:
+        if pointAis notNone:
```

Comment:
Here's an example of an invalid syntax.  Wouldn't it be quicker to just run a find and replace through the codebase?